tests/io-test.ts

```diff
@@ -100,6 +102,43 @@ describe('io', () => {
     });
   });
 
+  describe('writeTodoStorageFile', () => {
+    it('writes operations, joining with EOL and ending with EOL', () => {
+      const todoStorageFilePath = getTodoStorageFilePath(tmp);
+      const operations: Operation[] = [
+        'add|eslint|no-prototype-builtins|25|21|25|35|da39a3ee5e6b4b0d3255bfef95601890afd80709|1637107200000|||app/controllers/settings.js',
+        'add|eslint|no-prototype-builtins|26|19|26|33|da39a3ee5e6b4b0d3255bfef95601890afd80709|1637107200000|||app/controllers/settings.js',
+      ];
+
+      writeTodoStorageFile(todoStorageFilePath, operations);
+
+      const todoContents = readFileSync(todoStorageFilePath, { encoding: 'utf8' });
+      expect(todoContents).toEqual(operations[0] + EOL + operations[1] + EOL)
```

Comment:
Can we just make a test util for this, which will join the operations the way we want? We seem to use this below as well. The function could just take a single array, and you can concat arrays when needing to merge those arrays before joining.

Example:

```js
function joinOperations(...operations: (Operation[] | Operation)[]): string {
    const flatOperations = ([] as Operation[]).concat(...args);
    return flatOperations.join(EOL) + EOL;
}
```

Reply:
Sure thing! It's late here at the moment, but I should have some time tomorrow to implement this

Reply:
Thanks, this PR should be all setup to allow commits from maintainers 🤗